src/app/copy/inline-copy/inline-copy.component.less

```diff
@@ -0,0 +1,29 @@
+.pfng-inline-copy {
+  display: inline-flex;
+  border: 1px solid #bbb;
+  background-color: #fff;
+  font-size: smaller;
+  margin-left: .25em;
+  margin-right: .25em;
+  max-width: 100%;
+  &-txt-cont {
+    flex-grow: 1;
+    display: inline;
+    align-items: center;
```

Comment:
`display: inline;` looks unnecessary `.pfng-inline-copy-txt-cont` and `align-items: center;` doesn't do anything since it isn't a flex parent.